packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -78,6 +78,11 @@ export class Runtime {
    */
   public static readonly NODEJS_12_X = new Runtime('nodejs12.x', RuntimeFamily.NODEJS, { supportsInlineCode: true });
 
+  /**
+   * The NodeJS 14.x runtime (nodejs14.x)
+   */
+  public static readonly NODEJS_14_X = new Runtime('nodejs14.x', RuntimeFamily.NODEJS, { supportsInlineCode: true });
```

Comment:
```suggestion
  public static readonly NODEJS_14_X = new Runtime('nodejs14.x', RuntimeFamily.NODEJS, { supportsInlineCode: false });
```

I have tested this, and as of now the runtime does not support inline code in `eu-central-1`

> 8/17 | 8:24:49 AM | CREATE_FAILED        | AWS::Lambda::Function       | Lambda (LambdaD247545B) ZipFile can only be used when Runtime is set to either of nodejs8.10, nodejs10.x, nodejs12.x, python2.7, python3.6, python3.7, python3.8.